packages/markdown-graph/tsconfig.json

```diff
@@ -15,9 +15,15 @@
     "moduleResolution": "Bundler",
```

Comment:
This entire file needs to be changed to extend the file at /config/tsconfig.base.json